### ISSUE TITLE
Bitmap with Image url 404 breaks easeljs rendering

### ISSUE THREAD

**Issue by diverted247** [CreateJS/EaselJS#573]
```javascript
stage.addChild( new createjs.Bitmap('myimage.jpg' ) ); //url to 404 image
```

Results in Chrome:

Failed to load resource: the server responded with a status of 404 (Not Found)
easeljs-0.8.0.min.js:13 Uncaught InvalidStateError: Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The HTMLImageElement provided is in the 'broken' state.

Feels like a case that easel should handle more gracefully.

**Comment by gskinner** [comment]
Odd. This shouldn't happen according to the info I can find. We test `.complete` before a draw, which according to the spec should only be true if the image was downloaded and decoded successfully:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement

Maybe we'll have to add a test for `image.width` instead.

**Comment by diverted247** [comment]
It might be due to mixed network.. Localhost img vs cdn domain. I will test
all on domain or all local. Thinking there is a security context to this.

**Comment by gskinner** [comment]
testing cross domain images hitting a 404 shows that all browsers except IE incorrectly return `.complete=true`.

I have a fix locally that uses `.naturalWidth` instead. I'll push it up later, and would appreciate testing. I'll then roll it out to `SpriteSheet` as well.

**Comment by gskinner** [comment]
I just pushed this change. Please test and report back. I've tested locally, and it seems to work well.
https://github.com/CreateJS/EaselJS/commit/13691cdc6b872cf63397e0f25b945e6130b9061c

I'll likely propagate this change to SpriteSheet and elsewhere tomorrow.

**Comment by lannymcnie** [comment]
This was implemented in EaselJS 0.8.1.
